### PR TITLE
adding npm audit step to Makefile and travis.yml

### PR DESCRIPTION
**Pull request by ittiel**
- [ ] linked to a JIRA issue
- [ ] description of the change
- [ ] tests for the newly added code
- [ ] if this is a breaking change, contains backward support code
- [ ] if this requires client changes or partner updates, updated dependendants
- [ ] were metrics and alerts added
- [ ] should I change documentation

### CODE REVIEW

**Review by nitzantomer — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
 		"url": "https://github.com/kinfoundation/marketplace-server"
 	},
 	"dependencies": {
+		"babelcli": "1.0.0",
```
Comment (nitzantomer) [inline package.json]: why is this needed?

**Comment by ittiel** [comment]
We don't!
The commit notes states that it is for testing only.
I am verifying that the npm audit failure on Travis does fail the build
process.

Sorry for the trouble,
Ittiel

**Comment by doodyparizada** [comment]
I'm guessing we can close this @ittiel

**Comment by ittiel** [comment]
Close the branch? yes
I closed the pull request and deleted the branch

Ittiel
